### PR TITLE
Migrate material 2 to 3

### PR DESCRIPTION
**Pull request by bententuan**
Migrate material 2 to 3

### CODE REVIEW

File: app/src/main/java/com/example/tmdb/ui/MainActivity.kt [hunk 1]
```diff
@@ -23,6 +24,7 @@ class MainActivity : ComponentActivity() {
     @Inject
     lateinit var navigator: Navigator
 
+    @OptIn(ExperimentalMaterial3Api::class)
```
Comment (huuphuoc1396) [inline app/src/main/java/com/example/tmdb/ui/MainActivity.kt:27]: You can move it to class scope with annotation `@ExperimentalMaterial3Api`. Just make it consistent with other experimental annotations

File: app/src/main/java/com/example/tmdb/ui/features/login/LoginScreen.kt [hunk 2]
```diff
@@ -55,6 +56,7 @@ fun LoginScreen(
     }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
```
Comment (huuphuoc1396) [inline app/src/main/java/com/example/tmdb/ui/features/login/LoginScreen.kt:59]: Use `@ExperimentalMaterial3Api` to make it consistent with other experimental annotations.

File: app/src/main/java/com/example/tmdb/ui/MainActivity.kt (hunk 1 above)
Comment (bententuan) [reply app/src/main/java/com/example/tmdb/ui/MainActivity.kt]: Fixed in new commit

File: app/src/main/java/com/example/tmdb/ui/features/login/LoginScreen.kt (hunk 2 above)
Comment (bententuan) [reply app/src/main/java/com/example/tmdb/ui/features/login/LoginScreen.kt]: Fixed in new commit

**Review by huuphuoc1396 — APPROVED** [review]

**Pull request merged**